documentation/slate/source/includes/apis.md

```diff
@@ -1049,7 +1049,7 @@ In this example, the stop points within the circle (SP1, SP2 et SP3) can be reac
 The journeys can only use allowed vehicle journeys (as present in the `public_transport` or `on_demand_transport` sections).
 They also can only use the allowed stop points for getting in or out of a vehicle (as present in the `street_network`, `transfer` and `crow_fly` sections).
 
-For filtering vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used. 
+For filtering vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used.
```

Comment:
Is that correct? I know you didn't write it but shouldn't it be 'To filter vehicle journeys' ?

Reply:
```suggestion
To filter vehicle journeys, the identifier of a line, route, commercial mode, physical mode or network can be used.
```